mlperf_logging/compliance_checker/1.0.0/closed_rnnt.yaml

```diff
@@ -67,7 +67,7 @@
 - KEY:
     NAME: opt_gradient_clip_norm
     REQ:   EXACTLY_ONE
-    CHECK: " v['value'] == None "
+    CHECK: " v['value'] == 1 "
```

Comment:
shouldn't the value of `Inf` also be captured here?

Reply:
Right, fixed with https://github.com/mlcommons/logging/pull/95/commits/b200c5850dcc799236aef15c9010bff4ec522b55